cmd/meroxa/turbine_cli/utils.go

```diff
@@ -17,17 +17,25 @@ import (
 )
 
 type AppConfig struct {
-	Name     string `json:"name"`
-	Language string `json:"language"`
+	Name        string            `json:"name"`
+	Environment string            `json:"environment"`
```

Comment:
Maybe add it once we have Environment support?

Reply:
i just didn't want to be overwriting app.json in such a way that i would lose anything that was already there

Reply:
@janelletavares I understand. I feel like that shouldn't be there either, and only add it when we enable support to environments, but I won't become a blocker just for that. I don't think it's that important so feel free to ignore my suggestion.